regparser/grammar/terms.py

```diff
@@ -26,10 +26,15 @@
     LineStart()
     + Optional(Suppress(unified.any_depth_p))
     + e_tag.setResultsName("head")
+    + Suppress(ZeroOrMore(unified.any_depth_p))
     + ZeroOrMore(
         (atomic.conj_phrases + e_tag).setResultsName(
             "tail", listAllMatches=True))
+    + Suppress(ZeroOrMore(Regex(r",[a-zA-Z ]+,")))
```

Comment:
Yup. We have a definition that's like "Term, which something or other else, means..."